force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -6,7 +6,13 @@ public class CollectionsTasks {
   public static Integer countZerosInBitList(
     List<Integer> listWithZerosAndUnits
   ) {
-    return 0;
+    Integer counter = 0;
+    for (Integer i : listWithZerosAndUnits) {
+      if (i == 0) {
```

Comment:
Have you thought about this:
*can you implement it without checking the value/without if statement?

Reply:
Updated. I've done it without checking the value, but couldn't do it without IF statement